### PR TITLE
added logger

### PR DESCRIPTION
**Pull request by ilya449**

### CODE REVIEW

File: src/main/java/com/cinema/Main.java [hunk 1]
```diff
@@ -80,38 +82,38 @@ public static void main(String[] args) throws AuthenticationException {
...
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
 
         User registeredUser = authenticationService
                 .register("newUserEmail@mail.com", "!pass_*_pass!");
 
-        System.out.println("Trying to login user:");
-        System.out.println(authenticationService
+        logger.info("Trying to login user:");
+        logger.info(authenticationService
```
Comment (pavlokhomiak) [inline src/main/java/com/cinema/Main.java:95]: use try-catch

File: src/main/resources/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
+log4j.rootLogger=DEBUG, stdout, file
```
Comment (pavlokhomiak) [inline src/main/resources/log4j.properties:1]: use INFO level

**Review by pavlokhomiak — COMMENTED** [review]
good

File: src/main/resources/log4j.properties (hunk 2 above)
Comment (ilya449) [reply src/main/resources/log4j.properties:1]: I use not only .info(), I also use .error(), so I use DEBUG level

File: src/main/java/com/cinema/Main.java (hunk 1 above)
Comment (ilya449) [reply src/main/java/com/cinema/Main.java:95]: It is valid password, so here won't be any exceptions. But I changed message for clearer understanding, ty

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**